curr_scraper/apps/api_caller/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Currency(models.Model):
+    code = models.TextField()
```

Comment:
Dont use Text fields unless you really need to store multiline data in fields.
Switch to CharFields.

Also, make code unique